src/storage-adapters/git/index.js

```diff
@@ -13,12 +13,13 @@ import Git from './git.js';
 const fs = fsApi.promises;
 
 export default class GitAdapter {
-  constructor({ path, fileExtension, author, publish, prefixMessageToSnapshotId }) {
+  constructor({ path, author, publish, prefixMessageToSnapshotId }) {
     this.path = path;
     this.author = author;
-    this.fileExtension = fileExtension;
     this.needsPublication = publish;
     this.prefixMessageToSnapshotId = prefixMessageToSnapshotId;
+
+    mime.define({ 'text/markdown': ['md'] }, true); // ensure extension for markdown files is `.md` and not `.markdown`
```

Comment:
Does this has to be in the constructor?
Seems to me it's more global and should be below imports